lua/plugins/banhammer.lua

```diff
@@ -14,7 +14,7 @@ local function markup_tempban(chat_id, user_id, time_value)
 	local markup = {inline_keyboard={
 		{--first line
 			{text = '-', callback_data = ('tempban:val:m:%s:%s'):format(user_id, chat_id)},
-			{text = '🕑 '..time_value, callback_data = 'tempban:nil'},
+			{text = 'ðŸ•‘ '..time_value, callback_data = 'tempban:nil'},
```

Comment:
Missing emoji ⏱

Reply:
Uh, true. He said he can't fix it now. I will fix it soon